source/deployment/elasticsearch.rst

```diff
@@ -64,10 +64,12 @@ Limitations
 
   - "a", "an", "and", "are", "as", "at", "be", "but", "by", "for", "if", "in", "into", "is", "it", "no", "not", "of", "on", "or", "such", "that", "the", "their", "then", "there", "these", "they", "this", "to", "was", "will", "with"  
 
-2. Searching stop words in quotes returns more results than just the searched terms (`ticket <https://mattermost.atlassian.net/browse/PLT-7314>`__).
+2. Searching stop words in quotes returns more results than just the searched terms (`ticket <https://mattermost.atlassian.net/browse/MM-7216>`__).
 
 3. AWS Elasticsearch implementations have a limit of 1000 days of post history that is searchable.
 
+4. Search results are limited to a user's team and channel membership. This is enforced by the Mattermost server. The entities are indexed in Elasticsearch in a way that allows Mattermost to filter them when querying, so the Mattermost server narrows down the results on every Elasticsearch request applying those filters.
```

Comment:
This was suggested as an FAQ entry but I added it here as the content seemed to fit better. Happy to move it the the FAQs if it can be framed as such.